releasenotes/releasenotes.txt

```diff
@@ -1,5 +1,72 @@
 @RV@ @RC@
 
+Headline Features of Exiv2 v0.27.4
+----------------------------------
+
+Dedicated to the memory of our beautiful cat Lizzie who was put to sleep on 2021-02-13.
+
+Thank You to the following who have contributed to this release.
+
+| Contributor  | Activity                                          |
+|:-------------|:--------------------------------------------------|
+| Alex         | Project Management                                |
+| Arnold       | Keeps me honest!                                  |
+| Christoph    | Nikon and Canon Support                           |
+| David        | Help with legal investigation concerning bmff     |
+| Freddie      | Fuji support                                      |
+| Leo          | New python test scripts                           |
+| Milos        | DNG, easyaccess support                           |
+| Peter K      | bmff code                                         |
+| Peter S      | langAltValue.read() fix and helpful issue reports |
+
+1) Bug and security fixes:
+   - bmff support (.CR3, .HEIC and .AVIF).
```

Comment:
Maybe we should be explicit it is read support only?

Reply:
Yes.  I could explicitly say that.  As you know, that detail is documented in man/man1/exiv2.1

At the start of the bmff project, the design was to add code to jp2image.cpp for kinds of other bmff files.  For practical reasons, Peter opened bmffimage.cpp.  I was looking at the man page this morning and thought "Ah yes, we have read-write support for JP2, and read-only for the new formats.".  So, it's probably better and simpler to have them in different image handlers.

Reply:
https://github.com/Exiv2/exiv2/pull/1498/commits/08b5eb6136b999603b8dd340276fd214eb62bd7a